charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -281,7 +281,22 @@ data:
     # single namespace mode. Fluentd requires a ClusterRole to
     # enrich logs with kubernetes metadata. In a single namespace environment,
     # we're likely not going to get a ClusterRole from admins.
-    {{- if not .Values.global.singleNamespace }}
+    {{- if .Values.global.custom_logging.enabled }}
+
+      elasticsearch:
+        enabled: true
+        connection:
+          host: {{ printf "%s-external-es-proxy.%s" .Release.Name .Release.Namespace }}
+          port: 9200
+
+    elasticsearch:
+      enabled: true
+      client:
+        host: {{ printf "%s-external-es-proxy:9201" .Release.Name }}
+        log: error
+    {{- end }}
+
+    {{- if and (not .Values.global.singleNamespace ) (not .Values.global.custom_logging.enabled ) }}
       elasticsearch:
         enabled: true
         connection:
```

Comment:
Can we break this up into two `if` blocks that don't span two different levels of yaml? The way it is now is confusing. Having fewer conditionals is not as important here as readability. Let's try to make this as easy as possible to understand, even if there are redundant conditions.

Reply:
I still think this should be done.